index.js

```diff
@@ -61,6 +61,7 @@ Compare.prototype = {
     this._container.style.WebkitTransform = pos;
     this._clippedMap.getContainer().style.clip = 'rect(0, 999em, ' + this._bounds.height + 'px,' + x + 'px)';
     this._x = x;
+    this.currentPosition = this._x;
```

Comment:
Can `this.currentPosition` just replace `this._x`? I don't two named variables of this value is necessary?